buildpipeline/DotNet-Trusted-Publish.json

```diff
@@ -574,17 +574,11 @@
   "quality": "definition",
   "drafts": [],
   "queue": {
-    "_links": {
-      "self": {
-        "href": "https://devdiv.visualstudio.com/DefaultCollection/_apis/build/Queues/330"
-      }
-    },
-    "id": 330,
-    "name": "DotNetCore-Build",
-    "url": "https://devdiv.visualstudio.com/DefaultCollection/_apis/build/Queues/330",
+    "id": 36,
+    "name": "DotNet-Build",
     "pool": {
-      "id": 97,
-      "name": "DotNetCore-Build"
+      "id": 39,
+      "name": "DotNet-Build"
```

Comment:
While this is fine we probably should just update the definition to work with VS 2017, as I don't like the idea of multiple machine pools being necessary. That can happen independently.

Reply:
Yes, that's my expectation, but unbreaking the builds is p0